src/__tests__/suggestions.ts

```diff
@@ -238,7 +241,7 @@ it('should not suggest by label when using by label', async () => {
   )
 
   // if a suggestion is made, this call will throw, thus failing the test.
-  const password = await screen.findByLabelText(/bar/i)
+  const password = (await screen.findByLabelText(/bar/i)) as Element
```

Comment:
question: Why do we need this type casting?

Reply:
we don't have already converted findByLabelText so it will return an any type for now

Reply:
Could you do a 
```suggestion
  const password: Element = await screen.findByLabelText(/bar/i)
```
so that we're still safe once we type findByLabelText?

Reply:
it will still remain a warning, maybe we can leave as is